### PR TITLE
Release 0.6.0

### PR DESCRIPTION
**Pull request by duritong**
Also I added copr project site, I can add others to it if they have an account, so we can build things in a shared effort.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ Tries to follow the [packaging guidelines](https://fedoraproject.org/wiki/Packag
 * Sysconfig: `/etc/sysconfig/consul`
 * WebUI: `/usr/share/consul/`
 
+# Using
+
+Either you can build the RPMs on your own using the SPEC file in this repository
```
Comment (phrawzty) [inline README.md]: # Using

Create the RPMs using one of the techniques outlined in the Build section below.

## Pre-built packages

Pre-built packages are maintained via the [Fedora Copr](https://copr.fedoraproject.org/coprs/) system. For more information, please see the [duritong/consul](https://copr.fedoraproject.org/coprs/duritong/consul/) repository on Copr.

File: README.md [hunk 2]
```diff
@@ -8,6 +8,13 @@ Tries to follow the [packaging guidelines](https://fedoraproject.org/wiki/Packag
...
+or you can use the builded RPMs hosted on copr by one of the maintainers.
+
+See the [copr-project site](https://copr.fedoraproject.org/coprs/duritong/consul/) for more information on how to use the repository.
+
 # Build
```
Comment (phrawzty) [inline README.md]: I would move the "manual" build process to the top of this section since it's the most common denominator.

File: README.md [hunk 3]
```diff
@@ -8,6 +8,13 @@ Tries to follow the [packaging guidelines](https://fedoraproject.org/wiki/Packag
...
 
 If you have Vagrant installed:
```
Comment (phrawzty) [inline README.md]: ## Vagrant

File: SPECS/consul.spec [hunk 4]
```diff
@@ -53,7 +53,7 @@ mkdir -p %{buildroot}/%{_sysconfdir}/sysconfig
 cp %{SOURCE1} %{buildroot}/%{_sysconfdir}/sysconfig/%{name}
 mkdir -p %{buildroot}/%{_sharedstatedir}/%{name}
 mkdir -p %{buildroot}/%{_datadir}/%{name}-ui
-cp -r dist/* %{buildroot}/%{_prefix}/share/%{name}-ui
+cp -r index.html static %{buildroot}/%{_prefix}/share/%{name}-ui
```
Comment (phrawzty) [inline SPECS/consul.spec]: This changed in 0.6.0 ?

**Comment by phrawzty** [comment]
Hey mate, thanks for the bump. :smile: My documentation suggestions aren't critical - I'm just trying to keep it readable since the README is starting to get longer. This will be important if we add other build methods (like docker in #22), for example.

File: SPECS/consul.spec (hunk 4 above)
Comment (duritong) [inline SPECS/consul.spec]: yeah unfortunately. They seem to have a very consistent packaging process :/

**Comment by duritong** [comment]
I'm fine with adapting the README according to your changes. Will push an updated README.

**Comment by duritong** [comment]
So I integrated your comments

**Comment by phrawzty** [comment]
:+1:

**Pull request merged**